recipe/bld.bat

```diff
@@ -1,21 +1,15 @@
 REM edited from the one from Anaconda
 
 REM Build step
-nmake -f makefile.msc
-jom -f makefile.msc lib bzip2
-if errorlevel 1 exit 1
+nmake -f makefile.msc lib || exit 1
 
 REM Install step
-copy libbz2.lib %LIBRARY_LIB%\libbz2_static.lib || exit 1
-REM Some packages expect 'bzip2.lib', so make another copy
-copy libbz2.lib %LIBRARY_LIB%\bzip2_static.lib || exit 1
 copy bzlib.h %LIBRARY_INC% || exit 1
-
-cl /MD /O2 /Ibzlib.h /Ibzlib_private.h /D_USRDLL /D_WINDLL blocksort.c bzlib.c compress.c crctable.c decompress.c huffman.c randtable.c /LD /Felibbz2.dll /link /DEF:libbz2.def
-if errorlevel 1 exit 1
-
+copy libbz2_static.lib %LIBRARY_LIB% || exit 1
 copy libbz2.lib %LIBRARY_LIB% || exit 1
+copy libbz2.dll %LIBRARY_BIN% || exit 1
+
 REM Some packages expect 'bzip2.lib', so make copies
+copy libbz2_static.lib %LIBRARY_LIB%\bzip2_static.lib || exit 1
 copy libbz2.lib %LIBRARY_LIB%\bzip2.lib || exit 1
-copy libbz2.dll %LIBRARY_BIN% || exit 1
 copy libbz2.dll %LIBRARY_BIN%\bzip2.dll || exit 1
```

Comment:
This is not needed right?

Reply:
You mean the `bzip2.dll` specifically, right?
All of those have been grandfathered in from Anaconda's builds (partially, even before `conda.builder` was a thing) and are most likely of limited use.
But yeah, for `bzip2.dll` I can't see any sane reason since the references will all be to `LIBBZ2.dll` anyway.
```suggestion
```